plugins/inputs/opcua/opcua_test.go

```diff
@@ -129,6 +129,7 @@ func TestReadClientIntegration(t *testing.T) {
 				AuthMethod:     "Anonymous",
 				ConnectTimeout: config.Duration(10 * time.Second),
 				RequestTimeout: config.Duration(1 * time.Second),
+				SessionTimeout: config.Duration(1 * time.Minute),
```

Comment:
Keep the test as this, i.e. please remove this change!

Reply:
done